contracts/bls/BLSAccount.sol

```diff
@@ -23,7 +23,7 @@ contract BLSAccount is SimpleAccount, IBLSAccount {
 
     function initialize(uint256[4] memory aPublicKey) public virtual initializer {
         super._initialize(address(0));
-        publicKey = aPublicKey;
+        setBlsPublicKey(aPublicKey);
```

Comment:
I don't understand somthing. Line 25 seems to set the owner to `address(0)`? And `setBlsPublicKey` is `onlyOwner`? How does it work?

Reply:
Becausae onlyOwner was modified to allow `address(this)` as you can see. A name change for  onlyOwner should probably also be done. But not in this PR.